src/main.ts

```diff
@@ -26,10 +26,10 @@ const createRecipesInput = () => {
 }
 
 const showIngredients = () => {
-  const receivedIngredients = Storage.get('ingredients')
+  const receivedIngredients: string[] = Storage.get('ingredients')
   const $ingredientList = $('<div>').addClass('content__ingredients-list').appendTo('.content__list')
   
-  receivedIngredients.forEach(ingredient => {
+  receivedIngredients.forEach((ingredient: string) => {
```

Comment:
receivedIngredients is marked as Array<string>, so no need to write type of `ingredient` once again. TypeScript will know its a string